CodeListLibrary_project/clinicalcode/db_utils.py

```diff
@@ -1632,6 +1678,67 @@ def getHistoryTags(concept_id, concept_history_date):
         return [dict(list(zip(columns, row))) for row in cursor.fetchall()]
 
 
+def getHistoryTags_PhenotypeWorkingset(workingset_id, workingset_history_date):
+    ''' Get historic tags attached to a workingset that were effective from a point in time '''
+
+    my_params = {
+        'workingset_id': workingset_id,
+        'workingset_history_date': workingset_history_date
```

Comment:
This function is not needed as tags now are a columns, not a separate model

Reply:
This was an erroneous addition - I forgot to remove it. We're still in draft so won't be added for PR - as of recent commit, this is no longer present 👍